src/api/groups.js

```diff
@@ -1,8 +1,21 @@
-import { GROUPS } from 'test/mocks/groups';
+import { GROUPS_OK_PAGE_1, GROUPS_OK_PAGE_2 } from 'test/mocks/groups';
+import throttle from 'util/throttle';
 
-// eslint-disable-next-line
-export const getApiGroups = () => (
-  new Promise((resolve) => {
-    resolve(GROUPS);
-  })
-);
+export const getGroups = (page, params) => new Promise((resolve) => {
```

Comment:
Needs to be aligned with the new way we manage API and API tests (see https://github.com/entando/designtime-admin-app/pull/54)